### ISSUE TITLE
default signature by group

### ISSUE THREAD

**Issue by diemelone** [nextcloud/mail#4598]
Ability to set default signatures by groupadmins, with variables such as "group" "user" ...

Why? Getting it right for every user is quite hard.

Should look like this:
> Kind regards
> %user%
> Link to website

**Comment by ChristophWurst** [comment]
Makes sense for enterprise setups.

**Comment by diemelone** [comment]
and for social clubs, so it makes much sense for the standard edition too.

**Comment by ChristophWurst** [comment]
this has nothing to do with editions. replace *enterprise* with *organizations* if it makes more sense.

**Comment by diemelone** [comment]
So you were not referring to a plugin feature specifically for the nc enterprise version (if that is even possible?). If so, sorry!

**Comment by ChristophWurst** [comment]
Nonono, we don't do closed source software. This is merely about who this feature is for and whom I expect to sponsor the development :wink:

**Comment by JPustkuchen** [comment]
Indeed, this would be a very helpful feature for multi-user environments in companies, clubs, etc. where you want to follow a corporate design and / or need to care for legal texts!

This should also include the option to (un)lock the signature for certain groups.
